Default (OSX).sublime-keymap

```diff
@@ -7,7 +7,7 @@
 
   // Jump to a dependency
   {
-    "keys": ["super+alt+right"],
+    "keys": ["super+alt+down"],
```

Comment:
cntrl-alt-right is a workspace command in linux.... yes all the different
os specific bindings are a pain and since they are outside of sublime a
pain to overrride.

On Wed, Mar 11, 2015 at 7:15 PM, Joel Kemp <notifications@github.com> wrote:

> In Default (OSX).sublime-keymap
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26273833>:
>
> > @@ -7,7 +7,7 @@
> >
> >    // Jump to a dependency
> >    {
> > -    "keys": ["super+alt+right"],
> > +    "keys": ["super+alt+down"],
>
> Unfortunately, that's already bound to Go To Definition.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26273833>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David